src/puppetlabs/puppetdb/command.clj

```diff
@@ -233,9 +233,9 @@
 
 (defmacro upon-error-throw-fatality
   [& body]
-  `(try+
+  `(try
     ~@body
-    (catch Throwable e#
+    (catch Exception e#
```

Comment:
Hah, nope, but I imagine that might be how Throwable got here in the first place...